features/api_token_info.feature

```diff
@@ -0,0 +1,151 @@
+Feature: api.u.pro.token_info.v1
+
+  @uses.config.contract_token
+  Scenario Outline: Get token info from api
+    Given a `<release>` `<machine_type>` machine with ubuntu-advantage-tools installed
+    When I run `pro api u.pro.token_info.v1 --args token=$behave_var{contract_token}` with sudo
```

Comment:
Let`s either turn this into a `non-root` test or add a new case where we run it as `non-root`

Reply:
[same as here](https://github.com/canonical/ubuntu-pro-client/pull/3262/files#r1724896035) - we do have a step to check `root` and `non-root` are the same